src/main/sensors/initialisation.c

```diff
@@ -24,6 +24,7 @@
 
 #include "platform.h"
 
+#include "drivers/usb_cdc_debug.h"
```

Comment:
This file should not have this code a part of it by default, you also left in some code that would make this fail to compile if you don't compile with this option enabled, the programmer using this to help debug should add this where they need it, feels like this is leftover from some debugging you were doing.